guides/source/association_basics.md

```diff
@@ -788,7 +788,23 @@ module MyApplication
 end
 ```
 
-This will work fine, because both the `Supplier` and the `Account` class are defined within the same scope. But the following will _not_ work, because `Supplier` and `Account` are defined in different scopes:
+This will work fine, because both the `Supplier` and the `Account` class are defined within the same scope. This is highly beneficial as it allows you to organize your models according to their scope. For example:
+
+```ruby
+# app/models/business/account.rb
+class Business::Account < ApplicationRecord
+  belongs_to :business
+end
+
+# app/models/business.rb
+class Business < ApplicationRecord
+  has_many :accounts
+end
+```
```

Comment:
Keeping the same nested examples makes things more consistent.
````suggestion
```ruby
# app/models/my_application/business/supplier.rb
module MyApplication
  module Business
    class Supplier < ApplicationRecord
      has_one :account
    end
  end
end

# app/models/my_application/business/account.rb
module MyApplication
  module Business
    class Account < ApplicationRecord
      belongs_to :supplier
    end
  end
end
```
````